### PR TITLE
Fixes #26236 - made disk check silent in sat_versions > 6.2

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

**Comment by iNecas** [comment]
@jameerpathan111 feedback from QE welcome here

**Comment by kgaikwad** [comment]
@iNecas, 
Updated PR with knowledge document link in warning message.

File: definitions/checks/disk/performance.rb [hunk 1]
```diff
@@ -38,11 +42,19 @@ def check_only_single_device?
...
       def dirs_to_check
         return DEFAULT_DIRS.first(1) if check_only_single_device?
+
         DEFAULT_DIRS
       end
 
       private
 
+      def warning_message
+        "\nWARNING: Before taking any updates make sure"\
```
Comment (iNecas) [inline definitions/checks/disk/performance.rb]: Alternative wording:

```
WARNING: Low disk speed might have negative impact on the system.
Please consult https://access.redhat.com/solutions/3397771 before proceeding
with further actions.
``

Or even better, how about @swadeley having a look and giving us a native speaker suggestion on how to best formulate the message :)

Comment (swadeley) [reply definitions/checks/disk/performance.rb]: WARNING: Low disk speed might have a negative impact on the system. See https://access.redhat.com/solutions/3397771 before proceeding.

Comment (iNecas) [reply definitions/checks/disk/performance.rb]: Short and clean. Thanks @swadeley

**Comment by jameerpathan111** [comment]
@kgaikwad @iNecas 

Tested this PR, It works as expected
```
[root@hp-moonshot-01-c09 foreman_maintain]# ./bin/foreman-maintain health check --label disk-performance
Running ForemanMaintain::Scenario::FilteredScenario
================================================================================
Check for recommended disk speed of pulp, mongodb, pgsql dir.: 
/ Finished                                                                      

Disk speed : 33 MB/sec                                                [WARNING]
Slow disk detected /var/lib/pulp mounted on /dev/mapper/rhel_hp--moonshot--01--c09-root.
             Actual disk speed: 33 MB/sec
             Expected disk speed: 60 MB/sec.
WARNING: Low disk speed might have a negative impact on the system.
See https://access.redhat.com/solutions/3397771 before proceeding
--------------------------------------------------------------------------------
Scenario [ForemanMaintain::Scenario::FilteredScenario] failed.

The following steps ended up in warning state:

  [disk-performance]

The steps in warning state itself might not mean there is an error,
but it should be reviews to ensure the behavior is expected
```

@kgaikwad can we change `reviews` to `reviewed`

**Pull request merged**